quinn-proto/src/tests/mod.rs

```diff
@@ -1914,6 +1932,32 @@ fn malformed_token_len() {
     );
 }
 
+#[test]
+fn loss_probe_requests_immediate_ack() {
+    let _guard = subscribe();
+    let mut pair = Pair::default();
+    let (client_ch, _) = pair.connect();
+    pair.drive();
+
+    let stats_after_connect = pair.client_conn_mut(client_ch).stats();
+
+    // Lose a ping
+    let default_mtu = mem::replace(&mut pair.mtu, 0);
+    pair.client_conn_mut(client_ch).ping();
+    pair.drive_client();
+    pair.mtu = default_mtu;
+
+    // Drive the connection further so a loss probe is sent and the ping is recovered
```

Comment:
Pings actually never get retransmitted (see https://www.rfc-editor.org/rfc/rfc9000.html#name-retransmission-of-informati), though it doesn't seem like that's important to this test in practice.

Reply:
Thanks for the clarification, I have updated the comment